app/models/person_upload.rb

```diff
@@ -0,0 +1,32 @@
+require 'virtus'
+class PersonUpload
+  include Virtus.model
+  extend ActiveModel::Naming
+  include ActiveModel::Conversion
+  include ActiveModel::Validations
+
+  attr_accessor :file, :group_id
+  attr_reader :import_count, :csv_errors
+
+  validates :file, presence: true
+  validates :group_id, presence: true
+
+  def initialize(*)
+    @csv_errors = []
+    super
+  end
+
+  def save
+    return nil unless valid?
+
+    groups = Group.where(id: group_id)
+    importer = CsvPeopleImporter.new(file.read, groups: groups)
+    @import_count = importer.import
+    @csv_errors = importer.errors
+    @import_count
```

Comment:
I think this is fine to merge as it is (pending specs). But reading this I was thinking it would be nice if there was some indication here that `@import_count` can be nil without having to dig into the code `CsvPeopleImporter#import`. This is while I was trying to confirm that the behaviour of `PersonUpload#save` is congruent with `ActiveRecord` i.e. if the thing you're calling it on has validation errors, it returns falsy (which it does here, as expected).